watertap/unit_models/gac.py

```diff
@@ -1058,52 +1075,190 @@ def initialize_build(
 
     # ---------------------------------------------------------------------
 
-    # TODO: def _get_performance_contents(self, time_point=0):
-    # TODO: def _get_stream_table_contents(self, time_point=0):
+    def _get_performance_contents(self, time_point=0):
+        var_dict = {}
+
```

Comment:
Typically what variables should we consider as `performance_contents`? I see you almost list all vairables in your model, but maybe some variables are specific inputs but not performance indices? Since this is also my TODO list of  UV model, I want to confirm with you.

Reply:
I chose variables that are relevant for interpreting the results of the model. This includes variables that are typically fixed, indicative of the size of the unit, indicative of the performance, or other important variables. Just so happened that this turned out to be a lot, but there are some lesser relevant variables that are not included.

Reply:
@luohezhiming You can report any variable, expression, and/or parameter values that you think would be of interest. These would then be displayed when using the `report()` method (e.g., `m.fs.unit.report()`).